harness/determined/deploy/gcp/terraform/modules/service_account/main.tf

```diff
@@ -1,4 +1,9 @@
 // Create Service Account
+locals {
+  service_account_email = google_service_account.service_account.email
+  service_account_id = "serviceAccount:${local.service_account_email}"
```

Comment:
It's weird to call it serveice account id because it's a concept that doesn't exist. Can you call it `resource_member`?

Reply:
Sounds good! I wasn't sure what to call it.